test/e2e/framework/node/wait.go

```diff
@@ -198,8 +198,8 @@ func waitListSchedulableNodes(c clientset.Interface) (*v1.NodeList, error) {
 	return nodes, nil
 }
 
-// waitListSchedulableNodesOrDie is a wrapper around listing nodes supporting retries.
-func waitListSchedulableNodesOrDie(c clientset.Interface) (*v1.NodeList, error) {
+// checkWaitListSchedulableNodes is a wrapper around listing nodes supporting retries.
+func checkWaitListSchedulableNodes(c clientset.Interface) (*v1.NodeList, error) {
 	nodes, err := waitListSchedulableNodes(c)
```

Comment:
After removing `OrDie` suffix, there were two naming `waitListSchedulableNodes` methods. So I changed this one to `checkWaitListSchedulableNodes`